Paper UI: build problem

Hey there,

I'm currently trying to set up Eclipse Smart Home so that I am able to programm a custom binder. I've not a lot of experience with Maven (beside some simple builds) and I have quiet a hard time to make a working build. The build process fails everytime at the creation of the Paper - UI. This is the error message I get:

> [INFO] Building Eclipse SmartHome Paper UI 0.10.0-SNAPSHOT            [219/245]
[INFO] ---------------------------[ eclipse-plugin ]---------------------------
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ org.eclipse.smarthome.ui.paper ---
[INFO] Deleting K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\target
[INFO] Deleting K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper (includes = [node_modules/**, npm_cache/**, npm_tmp/**, web/**], excludes = [.gitignore])
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:install-node-and-npm (npm clean) @ org.eclipse.smarthome.ui.paper ---
[INFO] Installing node version v10.4.0
[INFO] Copying node binary from C:\Users\Tristan\.m2\repository\com\github\eirslett\node\10.4.0\node-10.4.0-windows-x64.exe to K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node\node.exe
[INFO] Installed node locally.
[INFO] Installing npm version 6.1.0
[INFO] Unpacking C:\Users\Tristan\.m2\repository\com\github\eirslett\npm\6.1.0\npm-6.1.0.tar.gz into K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node\node_modules
[INFO] Installed npm locally.
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:npm (npm clean) @ org.eclipse.smarthome.ui.paper ---
[INFO] Running 'npm cache clean --force' in K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper
[WARNING] npm WARN using --force I sure hope you know what you are doing.
[INFO] 
[INFO] --- tycho-packaging-plugin:1.2.0:build-qualifier (default-build-qualifier) @ org.eclipse.smarthome.ui.paper ---
[INFO] The project's OSGi version is 0.10.0.201811131542
[INFO] 
[INFO] --- tycho-packaging-plugin:1.2.0:validate-id (default-validate-id) @ org.eclipse.smarthome.ui.paper ---
[INFO] 
[INFO] --- tycho-packaging-plugin:1.2.0:validate-version (default-validate-version) @ org.eclipse.smarthome.ui.paper ---
[INFO] 
[INFO] --- directory-maven-plugin:0.2:highest-basedir (directories) @ org.eclipse.smarthome.ui.paper ---
[INFO] Highest basedir set to: K:\Eclipse Installed\smarthome-master\git\smarthome
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:install-node-and-npm (Install node and npm) @ org.eclipse.smarthome.ui.paper ---
[INFO] Node v10.4.0 is already installed.
[INFO] NPM 6.1.0 is already installed.
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:npm (npm install) @ org.eclipse.smarthome.ui.paper ---
[INFO] Running 'npm install' in K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper
[WARNING] npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[WARNING] npm WARN deprecated openlayers@4.2.0: Please use the ol package instead
[WARNING] npm WARN deprecated gulp-util@2.2.20: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[WARNING] npm WARN deprecated nomnom@1.8.1: Package no longer supported. Contact support@npmjs.com for more info.
[WARNING] npm WARN deprecated graceful-fs@3.0.11: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
[WARNING] npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[WARNING] npm WARN deprecated express@2.5.11: express 2.x series is deprecated
[WARNING] npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
[WARNING] npm WARN deprecated connect@1.9.2: connect 1.x series is deprecated
[WARNING] npm WARN deprecated uws@9.14.0: stop using this version
[WARNING] npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[WARNING] npm WARN deprecated node-uuid@1.4.8: Use uuid module instead
[WARNING] npm WARN deprecated graceful-fs@1.2.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
[WARNING] npm WARN deprecated cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
[WARNING] npm WARN deprecated hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
[WARNING] npm WARN deprecated boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
[INFO] 
[INFO] > uws@9.14.0 install K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\uws
[INFO] > node-gyp rebuild > build_log.txt 2>&1 || exit 0
[INFO] 
[INFO] 
[INFO] > phantomjs-prebuilt@2.1.16 install K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\phantomjs-prebuilt
[INFO] > node install.js
[INFO] 
[INFO] PhantomJS not found on PATH
[INFO] Downloading https://github.com/Medium/phantomjs/releases/download/v2.1.1/phantomjs-2.1.1-windows.zip
[INFO] Saving to K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\npm_tmp\phantomjs\phantomjs-2.1.1-windows.zip
[INFO] Receiving...
[INFO] 
[INFO] Received 17767K total.
[INFO] Extracting zip contents
[INFO] Removing K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\phantomjs-prebuilt\lib\phantom
[INFO] Copying extracted folder K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\npm_tmp\phantomjs\phantomjs-2.1.1-windows.zip-extract-1542124138701\phantomjs-2.1.1-windows -> K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\phantomjs-prebuilt\lib\phantom
[INFO] Writing location.js file
[INFO] Done. Phantomjs binary available at K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\phantomjs-prebuilt\lib\phantom\bin\phantomjs.exe
[INFO] 
[INFO] > openlayers@4.2.0 install K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\openlayers
[INFO] > node tasks/install.js
[INFO] 
[INFO] 
[INFO] > closure-util@1.21.0 postinstall K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\closure-util
[INFO] > node ./bin/closure-util.js update
[INFO] 
[ERROR] info install Downloading https://github.com/google/closure-library/archive/v20170409.zip
[ERROR] info install Downloading https://dl.google.com/closure-compiler/compiler-20170521.zip
[ERROR] info install Download complete: K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\closure-util\.deps\compiler\3f3b5a5ee1025a8fe779da0edfe3e32f5cd20f2f
[ERROR] info install Download complete: K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\closure-util\.deps\library\0f95ad8150b77cd8db0e8d9e9726292ab33580af
[INFO] 
[INFO] > openlayers@4.2.0 postinstall K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\node_modules\openlayers
[INFO] > closure-util update
[INFO] 
[INFO] 
[INFO] > org.eclipse.smarthome.ui.paperui@0.10.0 install K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper
[INFO] > node node_modules/openlayers/tasks/build.js openlayers-esh.json node_modules/openlayers/ol-esh.js
[INFO] 
[ERROR] ERR! Trouble generating info: Der Befehl "K:\Eclipse" ist entweder falsch geschrieben oder
[ERROR] konnte nicht gefunden werden.
[ERROR]  
[ERROR] npm ERR! code ELIFECYCLE
[ERROR] npm ERR! errno 1
[ERROR] npm ERR! org.eclipse.smarthome.ui.paperui@0.10.0 install: `node node_modules/openlayers/tasks/build.js openlayers-esh.json node_modules/openlayers/ol-esh.js`
[ERROR] npm ERR! Exit status 1
[ERROR] npm ERR! 
[ERROR] npm ERR! Failed at the org.eclipse.smarthome.ui.paperui@0.10.0 install script.
[ERROR] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[ERROR] 
[ERROR] npm ERR! A complete log of this run can be found in:
[ERROR] npm ERR!     K:\Eclipse Installed\smarthome-master\git\smarthome\extensions\ui\org.eclipse.smarthome.ui.paper\npm_cache\_logs\2018-11-13T15_49_09_164Z-debug.log
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] Eclipse SmartHome 0.10.0-SNAPSHOT .................. SUCCESS [  0.182 s]
[INFO] Eclipse SmartHome Bundles .......................... SUCCESS [  0.198 s]
[INFO] Eclipse SmartHome Test Components .................. SUCCESS [  0.022 s]
[INFO] Eclipse SmartHome Core Components .................. SUCCESS [  0.017 s]
[INFO] Eclipse SmartHome Core ............................. SUCCESS [  4.891 s]
[INFO] Eclipse SmartHome Configurations ................... SUCCESS [  0.019 s]
[INFO] Eclipse SmartHome Configuration Core ............... SUCCESS [  1.274 s]
[INFO] Eclipse SmartHome Configuration XML ................ SUCCESS [  0.917 s]
[INFO] Eclipse SmartHome Test ............................. SUCCESS [  0.592 s]
[INFO] Eclipse SmartHome I/O Components ................... SUCCESS [  0.016 s]
[INFO] Eclipse SmartHome Console .......................... SUCCESS [  0.398 s]
[INFO] Eclipse SmartHome Thing ............................ SUCCESS [  2.401 s]
[INFO] Eclipse SmartHome Configuration Discovery .......... SUCCESS [  0.580 s]
[INFO] Eclipse SmartHome Network I/O ...................... SUCCESS [  0.384 s]
[INFO] Eclipse SmartHome Magic Bundle ..................... SUCCESS [  0.548 s]
[INFO] Eclipse SmartHome Magic Bundle Tests ............... SUCCESS [  0.457 s]
[INFO] Eclipse SmartHome Auth Components .................. SUCCESS [  0.020 s]
[INFO] Eclipse SmartHome JAAS Authentication .............. SUCCESS [  0.213 s]
[INFO] Eclipse SmartHome OAuth2Client ..................... SUCCESS [  0.405 s]
[INFO] Eclipse SmartHome Configuration Dispatcher ......... SUCCESS [  0.293 s]
[INFO] Eclipse SmartHome Configuration Dispatcher Tests ... SUCCESS [  0.412 s]
[INFO] Eclipse SmartHome Configuration Core Tests ......... SUCCESS [  0.497 s]
[INFO] Eclipse SmartHome mDNS Service ..................... SUCCESS [  0.253 s]
[INFO] Eclipse SmartHome Configuration mDNS Discovery ..... SUCCESS [  0.347 s]
[INFO] Eclipse SmartHome Core Thing XML ................... SUCCESS [  0.505 s]
[INFO] Eclipse SmartHome Configuration mDNS Discovery Tests SUCCESS [  0.294 s]
[INFO] Eclipse SmartHome Configuration Discovery Tests .... SUCCESS [  0.576 s]
[INFO] Eclipse SmartHome Configuration UPnP Discovery ..... SUCCESS [  0.180 s]
[INFO] Eclipse SmartHome Configuration USB-Serial Discovery SUCCESS [  0.201 s]
[INFO] Eclipse SmartHome Configuration Discovery UsbSerial Tests SUCCESS [  0.392 s]
[INFO] Eclipse SmartHome Configuration USB-Serial Discovery for Linux using sysfs scanning SUCCESS [  0.267 s]
[INFO] Eclipse SmartHome Serial Transport ................. SUCCESS [  0.266 s]
[INFO] Eclipse SmartHome Configuration Serial ............. SUCCESS [  0.151 s]
[INFO] Eclipse SmartHome Config XML Tests ................. SUCCESS [  0.409 s]
[INFO] Eclipse SmartHome HTTP Interface ................... SUCCESS [  0.287 s]
[INFO] Eclipse SmartHome Core Audio ....................... SUCCESS [  0.318 s]
[INFO] Eclipse SmartHome Core Binding XML ................. SUCCESS [  0.209 s]
[INFO] Eclipse SmartHome Audio Test ....................... SUCCESS [  0.352 s]
[INFO] Eclipse SmartHome Sample Extension Service ......... SUCCESS [  0.178 s]
[INFO] Eclipse SmartHome REST Interface ................... SUCCESS [  0.357 s]
[INFO] Eclipse SmartHome Core ID .......................... SUCCESS [  0.279 s]
[INFO] Eclipse SmartHome ID Tests ......................... SUCCESS [  0.216 s]
[INFO] Eclipse SmartHome Core Tests ....................... SUCCESS [  1.438 s]
[INFO] Eclipse SmartHome Thing Tests ...................... SUCCESS [  1.881 s]
[INFO] Eclipse SmartHome Core Persistence ................. SUCCESS [  0.656 s]
[INFO] Eclipse SmartHome Core Scheduler Service ........... SUCCESS [  0.159 s]
[INFO] Eclipse SmartHome Core Semantics ................... SUCCESS [  0.587 s]
[INFO] Eclipse SmartHome Core Transformation Service ...... SUCCESS [  0.305 s]
[INFO] Eclipse SmartHome Core Transform Test .............. SUCCESS [  0.274 s]
[INFO] Eclipse SmartHome Core Binding XML Tests ........... SUCCESS [  0.350 s]
[INFO] Eclipse SmartHome Core Thing XML Tests ............. SUCCESS [  0.516 s]
[INFO] Eclipse SmartHome Core Voice ....................... SUCCESS [  0.536 s]
[INFO] Eclipse SmartHome Core Voice Test .................. SUCCESS [  0.476 s]
[INFO] Eclipse SmartHome Automation ....................... SUCCESS [  0.027 s]
[INFO] Eclipse SmartHome Automation API ................... SUCCESS [  0.515 s]
[INFO] Eclipse SmartHome Automation Commands .............. SUCCESS [  0.397 s]
[INFO] Eclipse SmartHome Automation Core .................. SUCCESS [  0.735 s]
[INFO] Eclipse SmartHome Automation Parser GSON ........... SUCCESS [  0.234 s]
[INFO] Eclipse SmartHome Automation Providers ............. SUCCESS [  0.336 s]
[INFO] Eclipse SmartHome Automation Provider File ......... SUCCESS [  0.231 s]
[INFO] Eclipse SmartHome Automation Core Tests ............ SUCCESS [  0.359 s]
[INFO] Eclipse SmartHome Automation Sample JSON ........... SUCCESS [  0.225 s]
[INFO] Eclipse SmartHome Automation Core Modules .......... SUCCESS [  0.371 s]
[INFO] Eclipse SmartHome Automation Core Module Tests ..... SUCCESS [  0.474 s]
[INFO] Eclipse SmartHome Automation Script Modules ........ SUCCESS [  0.367 s]
[INFO] Eclipse SmartHome Automation Media Modules ......... SUCCESS [  0.310 s]
[INFO] Eclipse SmartHome Automation Timer Modules ......... SUCCESS [  0.241 s]
[INFO] Eclipse SmartHome Automation Module Timer Tests .... SUCCESS [  0.311 s]
[INFO] Eclipse SmartHome Automation Script Default Scope .. SUCCESS [  0.210 s]
[INFO] Eclipse SmartHome Automation Script Module Defaultscope Tests SUCCESS [  0.212 s]
[INFO] Eclipse SmartHome Automation Script RuleSupport .... SUCCESS [  0.465 s]
[INFO] Eclipse SmartHome Automation Script Module Tests ... SUCCESS [  0.262 s]
[INFO] Eclipse SmartHome Automation Event Tests ........... SUCCESS [  0.227 s]
[INFO] Eclipse SmartHome Automation Integration Tests ..... SUCCESS [  0.587 s]
[INFO] Eclipse SmartHome Automation Sample Extension Java . SUCCESS [  0.293 s]
[INFO] Eclipse SmartHome Automation REST API .............. SUCCESS [  0.262 s]
[INFO] Eclipse SmartHome Automation Sample REST API ....... SUCCESS [  0.180 s]
[INFO] Eclipse SmartHome Automation Java Demo ............. SUCCESS [  0.132 s]
[INFO] Eclipse SmartHome Automation Json Demo ............. SUCCESS [  0.058 s]
[INFO] Eclipse SmartHome Automation Module Type Demo ...... SUCCESS [  0.226 s]
[INFO] Eclipse SmartHome Console for OSGi framework Eclipse Equinox SUCCESS [  0.154 s]
[INFO] Eclipse SmartHome Console for OSGi Console RFC 147 . SUCCESS [  0.168 s]
[INFO] Eclipse SmartHome Console for OSGi runtime Karaf ... SUCCESS [  0.163 s]
[INFO] Eclipse SmartHome Monitor .......................... SUCCESS [  0.153 s]
[INFO] Eclipse SmartHome Network I/O Tests ................ SUCCESS [  0.322 s]
[INFO] Eclipse SmartHome HTTP Interface Authentication .... SUCCESS [  0.193 s]
[INFO] Eclipse SmartHome HTTP Interface Basic Authentication SUCCESS [  0.153 s]
[INFO] Eclipse SmartHome Network I/O HTTP Tests ........... SUCCESS [  0.178 s]
[INFO] Eclipse SmartHome IO REST Tests .................... SUCCESS [  0.291 s]
[INFO] Eclipse SmartHome Authentication Support for the REST Interface SUCCESS [  0.159 s]
[INFO] Eclipse SmartHome Core REST Interface .............. SUCCESS [  0.759 s]
[INFO] Eclipse SmartHome IO REST Core Tests ............... SUCCESS [  0.414 s]
[INFO] Eclipse SmartHome REST mDNS Announcer .............. SUCCESS [  0.151 s]
[INFO] Eclipse SmartHome REST JAX-RS Optimizations ........ SUCCESS [  0.203 s]
[INFO] Eclipse SmartHome SSE Interface .................... SUCCESS [  0.475 s]
[INFO] Eclipse SmartHome IO SSE Tests ..................... SUCCESS [  0.264 s]
[INFO] Eclipse SmartHome Models ........................... SUCCESS [  2.938 s]
[INFO] Eclipse SmartHome Model Core ....................... SUCCESS [  1.221 s]
[INFO] Eclipse SmartHome Model Lazy Generation ............ SUCCESS [  0.493 s]
[INFO] Eclipse SmartHome Model Sitemap .................... SUCCESS [ 15.381 s]
[INFO] Eclipse SmartHome Model Items ...................... SUCCESS [  6.823 s]
[INFO] Eclipse SmartHome UIs .............................. SUCCESS [  0.018 s]
[INFO] Eclipse SmartHome UI ............................... SUCCESS [  0.536 s]
[INFO] Eclipse SmartHome Sitemap REST Interface ........... SUCCESS [  0.497 s]
[INFO] Eclipse SmartHome Sitemap REST Interface Tests ..... SUCCESS [  0.317 s]
[INFO] Eclipse SmartHome Core REST UI Logging Module ...... SUCCESS [  0.201 s]
[INFO] Eclipse SmartHome Voice REST Interface ............. SUCCESS [  0.313 s]
[INFO] Eclipse SmartHome DBus Transport ................... SUCCESS [  0.111 s]
[INFO] Eclipse SmartHome MQTT Transport ................... SUCCESS [  0.450 s]
[INFO] Eclipse SmartHome Mqtt Transport I/O Tests ......... SUCCESS [  0.322 s]
[INFO] Eclipse SmartHome Serial Transport for Java Communications API SUCCESS [  0.267 s]
[INFO] Eclipse SmartHome Serial Transport for RXTX ........ SUCCESS [  0.265 s]
[INFO] Eclipse SmartHome Serial Transport for RFC2217 ..... SUCCESS [  0.242 s]
[INFO] Eclipse SmartHome UPnP Transport ................... SUCCESS [  0.283 s]
[INFO] Eclipse SmartHome IO UPnP Tests .................... SUCCESS [  0.200 s]
[INFO] Eclipse SmartHome Model Core Tests ................. SUCCESS [  0.569 s]
[INFO] Eclipse SmartHome Model Item IDE ................... SUCCESS [  1.231 s]
[INFO] Eclipse SmartHome Model Items Runtime .............. SUCCESS [  0.384 s]
[INFO] Eclipse SmartHome Items Model Tests ................ SUCCESS [  0.709 s]
[INFO] Eclipse SmartHome Model Persistence ................ SUCCESS [  6.578 s]
[INFO] Eclipse SmartHome Model Persistence IDE ............ SUCCESS [  0.867 s]
[INFO] Eclipse SmartHome Model Script ..................... SUCCESS [ 11.383 s]
[INFO] Eclipse SmartHome Model Rules ...................... SUCCESS [ 13.612 s]
[INFO] Eclipse SmartHome Model Rule IDE ................... SUCCESS [  1.813 s]
[INFO] Eclipse SmartHome Model Script IDE ................. SUCCESS [  1.581 s]
[INFO] Eclipse SmartHome Model Sitemap IDE ................ SUCCESS [  1.589 s]
[INFO] Eclipse SmartHome Thing Model ...................... SUCCESS [  7.617 s]
[INFO] Eclipse SmartHome Model Thing IDE .................. SUCCESS [  0.877 s]
[INFO] Eclipse SmartHome Language Server .................. SUCCESS [  0.714 s]
[INFO] Eclipse SmartHome Language Server Tests ............ SUCCESS [  0.481 s]
[INFO] Eclipse SmartHome Model Persistence Runtime ........ SUCCESS [  0.390 s]
[INFO] Eclipse SmartHome Model Persistence Tests .......... SUCCESS [  0.440 s]
[INFO] Eclipse SmartHome Model Rules Runtime .............. SUCCESS [  0.781 s]
[INFO] Eclipse SmartHome Model Script Runtime ............. SUCCESS [  0.477 s]
[INFO] Eclipse SmartHome Model Rules Tests ................ SUCCESS [  0.549 s]
[INFO] Eclipse SmartHome Model Script Tests ............... SUCCESS [  0.756 s]
[INFO] Eclipse SmartHome Model Sitemap Runtime ............ SUCCESS [  0.395 s]
[INFO] Eclipse SmartHome Thing Model Runtime .............. SUCCESS [  0.419 s]
[INFO] Eclipse SmartHome Thing Model Tests ................ SUCCESS [  0.930 s]
[INFO] Eclipse SmartHome UI Tests ......................... SUCCESS [  0.317 s]
[INFO] Eclipse SmartHome UI Icon Support .................. SUCCESS [  0.318 s]
[INFO] Eclipse SmartHome UI Icon Tests .................... SUCCESS [  0.383 s]
[INFO] Eclipse SmartHome Storages ......................... SUCCESS [  0.015 s]
[INFO] Eclipse SmartHome JSON Storage ..................... SUCCESS [  0.324 s]
[INFO] Eclipse SmartHome Storage JSON Tests ............... SUCCESS [  0.314 s]
[INFO] Eclipse SmartHome MapDB Storage .................... SUCCESS [  0.201 s]
[INFO] Eclipse SmartHome Storage MapDB Tests .............. SUCCESS [  0.232 s]
[INFO] Eclipse SmartHome Extensions ....................... SUCCESS [  0.015 s]
[INFO] Eclipse SmartHome Bindings ......................... SUCCESS [  0.013 s]
[INFO] Eclipse SmartHome Astro Binding .................... SUCCESS [  0.556 s]
[INFO] Eclipse SmartHome Astro Binding Tests .............. SUCCESS [  0.493 s]
[INFO] Eclipse SmartHome Bluetooth Binding ................ SUCCESS [  0.623 s]
[INFO] Eclipse SmartHome BlueGiga Bluetooth Adapter ....... SUCCESS [  1.114 s]
[INFO] Eclipse SmartHome Bluetooth BlueGiga Tests ......... SUCCESS [  0.185 s]
[INFO] Eclipse SmartHome BlueZ Bluetooth Adapter .......... SUCCESS [  0.283 s]
[INFO] Eclipse SmartHome Blukii Binding ................... SUCCESS [  0.181 s]
[INFO] Eclipse SmartHome Bluetooth Binding Tests .......... SUCCESS [  0.168 s]
[INFO] Eclipse SmartHome Bose SoundTouch Binding .......... SUCCESS [  0.407 s]
[INFO] Eclipse SmartHome DigitalSTROM Binding ............. SUCCESS [  2.084 s]
[INFO] Eclipse SmartHome DMX Binding ...................... SUCCESS [  0.427 s]
[INFO] Eclipse SmartHome DMX Binding Tests ................ SUCCESS [  0.616 s]
[INFO] Eclipse SmartHome Homematic Binding ................ SUCCESS [  2.235 s]
[INFO] Eclipse SmartHome Homematic Binding Tests .......... SUCCESS [  0.263 s]
[INFO] Eclipse SmartHome hue Binding ...................... SUCCESS [  0.588 s]
[INFO] Eclipse SmartHome hue Binding Tests ................ SUCCESS [  0.362 s]
[INFO] Eclipse SmartHome FSInternetRadio Binding .......... SUCCESS [  0.344 s]
[INFO] Eclipse SmartHome FSInternetRadio Binding Test ..... SUCCESS [  0.298 s]
[INFO] Eclipse SmartHome LIFX Binding ..................... SUCCESS [  2.648 s]
[INFO] Eclipse SmartHome LIRC Binding ..................... SUCCESS [  0.269 s]
[INFO] Eclipse SmartHome MQTT Binding ..................... SUCCESS [  0.374 s]
[INFO] Eclipse SmartHome MQTT Binding Tests ............... SUCCESS [  0.306 s]
[INFO] Eclipse SmartHome MQTT Thing Binding ............... SUCCESS [  0.853 s]
[INFO] Eclipse SmartHome I/O Services ..................... SUCCESS [  0.015 s]
[INFO] Eclipse SmartHome Embedded MQTT Broker ............. SUCCESS [  0.393 s]
[INFO] Eclipse SmartHome MQTT Thing Binding Tests ......... SUCCESS [  0.636 s]
[INFO] Eclipse SmartHome NTP Binding ...................... SUCCESS [  0.199 s]
[INFO] Eclipse SmartHome NTP Binding Tests ................ SUCCESS [  0.245 s]
[INFO] Eclipse SmartHome OneWire Binding .................. SUCCESS [  0.788 s]
[INFO] Eclipse SmartHome OneWire Binding Tests ............ SUCCESS [  0.844 s]
[INFO] Eclipse SmartHome SerialButton Binding ............. SUCCESS [  0.175 s]
[INFO] Eclipse SmartHome Sonos Binding .................... SUCCESS [  0.491 s]
[INFO] Eclipse SmartHome SonyAudio Binding ................ SUCCESS [  0.408 s]
[INFO] Eclipse SmartHome TRÅDFRI Binding .................. SUCCESS [  0.472 s]
[INFO] Eclipse SmartHome TRÅDFRI Binding Tests ............ SUCCESS [  0.235 s]
[INFO] Eclipse SmartHome WeatherUnderground Binding ....... SUCCESS [  0.389 s]
[INFO] Eclipse SmartHome WeMo Binding ..................... SUCCESS [  0.454 s]
[INFO] Eclipse SmartHome Wemo Binding Tests ............... SUCCESS [  0.391 s]
[INFO] Eclipse SmartHome YahooWeather Binding ............. SUCCESS [  0.241 s]
[INFO] Eclipse SmartHome Extension Service Extensions ..... SUCCESS [  0.014 s]
[INFO] Eclipse SmartHome IoT Marketplace Extension Service  SUCCESS [  0.401 s]
[INFO] Eclipse SmartHome IoT Marketplace Extension Automation Support SUCCESS [  0.215 s]
[INFO] Eclipse SmartHome IoT Marketplace Extension Service Test SUCCESS [  0.151 s]
[INFO] Eclipse SmartHome JavaSound I/O .................... SUCCESS [  0.255 s]
[INFO] Eclipse SmartHome Embedded MQTT Broker Tests ....... SUCCESS [  0.204 s]
[INFO] Eclipse SmartHome Web Audio Support ................ SUCCESS [  0.234 s]
[INFO] Eclipse SmartHome Persistence ...................... SUCCESS [  0.014 s]
[INFO] Eclipse SmartHome MapDB Persistence ................ SUCCESS [  0.261 s]
[INFO] Eclipse SmartHome MapDB Persistence Tests .......... SUCCESS [  0.247 s]
[INFO] Eclipse SmartHome Transformation Services .......... SUCCESS [  0.014 s]
[INFO] Eclipse SmartHome Exec Transformation Service ...... SUCCESS [  1.222 s]
[INFO] Eclipse SmartHome Javascript Transformation Service  SUCCESS [  0.193 s]
[INFO] Eclipse SmartHome Map Transformation Service ....... SUCCESS [  0.170 s]
[INFO] Eclipse SmartHome Map Transformation Service Tests . SUCCESS [  0.178 s]
[INFO] Eclipse SmartHome Scale Transformation Service ..... SUCCESS [  0.195 s]
[INFO] Eclipse SmartHome Scale Transformation Service Tests SUCCESS [  0.142 s]
[INFO] Eclipse SmartHome RegEx Transformation Service ..... SUCCESS [  0.176 s]
[INFO] Eclipse SmartHome RegEx Transformation Service Tests SUCCESS [  0.139 s]
[INFO] Eclipse SmartHome XPath Transformation Service ..... SUCCESS [  0.174 s]
[INFO] Eclipse SmartHome XPath Transformation Service Tests SUCCESS [  0.140 s]
[INFO] Eclipse SmartHome JSonPath Transformation Service .. SUCCESS [  0.196 s]
[INFO] Eclipse SmartHome JSonPath Transformation Service Tests SUCCESS [  0.150 s]
[INFO] Eclipse SmartHome Xslt Transformation Service ...... SUCCESS [  0.175 s]
[INFO] Eclipse SmartHome Xslt Transformation Service Tests  SUCCESS [  0.142 s]
[INFO] Eclipse SmartHome UI Extensions .................... SUCCESS [  0.014 s]
[INFO] Eclipse SmartHome Icon Sets ........................ SUCCESS [  0.014 s]
[INFO] Eclipse SmartHome Classic Icon Set ................. SUCCESS [  0.461 s]
[INFO] Eclipse SmartHome Basic UI ......................... SUCCESS [  9.260 s]
[INFO] Eclipse SmartHome Classic UI ....................... SUCCESS [  0.609 s]
[INFO] Eclipse SmartHome Paper UI ......................... FAILURE [02:37 min]
[INFO] Eclipse SmartHome Voice Services ................... SKIPPED
[INFO] Eclipse SmartHome macOS Text-to-Speech ............. SKIPPED
[INFO] Eclipse SmartHome macOS Text-to-Speech Tests ....... SKIPPED
[INFO] Eclipse SmartHome Features ......................... SKIPPED
[INFO] Eclipse SmartHome Runtime Core Feature ............. SKIPPED
[INFO] Eclipse SmartHome Runtime Model Feature ............ SKIPPED
[INFO] Eclipse SmartHome Runtime UI Feature ............... SKIPPED
[INFO] Eclipse SmartHome Runtime REST Feature ............. SKIPPED
[INFO] Eclipse SmartHome Runtime Automation Feature ....... SKIPPED
[INFO] Eclipse SmartHome Runtime Equinox Console Feature .. SKIPPED
[INFO] Eclipse SmartHome Runtime RfC147 Console Feature ... SKIPPED
[INFO] Eclipse SmartHome Runtime Binding Feature .......... SKIPPED
[INFO] Eclipse SmartHome Test Dependencies ................ SKIPPED
[INFO] Eclipse SmartHome Karaf Features ................... SKIPPED
[INFO] Eclipse SmartHome Target Platform Feature .......... SKIPPED
[INFO] Eclipse SmartHome Core Feature ..................... SKIPPED
[INFO] Eclipse SmartHome Extensions Feature ............... SKIPPED
[INFO] Eclipse SmartHome Karaf Feature Verification ....... SKIPPED
[INFO] Eclipse SmartHome Products ......................... SKIPPED
[INFO] Eclipse SmartHome p2 Repository .................... SKIPPED
[INFO] Eclipse SmartHome Tools ............................ SKIPPED
[INFO] Eclipse SmartHome Archetypes ....................... SKIPPED
[INFO] Eclipse SmartHome Binding Archetype ................ SKIPPED
[INFO] Eclipse SmartHome Binding Archetype Testsuite ...... SKIPPED
[INFO] Eclipse SmartHome Target Platform .................. SKIPPED
[INFO] Eclipse SmartHome Distribution 0.10.0-SNAPSHOT ..... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 06:51 min
[INFO] Finished at: 2018-11-13T16:49:09+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.0:npm (npm install) on project org.eclipse.smarthome.ui.paper: Failed to run task: 'npm install' failed. (error code 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :org.eclipse.smarthome.ui.paper

I srsly have no idea how i can solve my error. I tried countless advices from other posts, but nothing seems to work. For example: I already tried to clean my git workspace (#4863).
My operating system is Windows 7. I hope you can help me out.

I'll excuse in advance for my language and grammar mistakes, but english is not my native language :)

Greetings,
Tmirror

UPDATE:

I found the problem:

> [ERROR] ERR! Trouble generating info: Der Befehl "K:\Eclipse" ist entweder falsch geschrieben oder
[ERROR] konnte nicht gefunden werden.

Seems like there isn't a space in the name of the base folder allowed. I'm not sure if this is intended or even known. I'll leave this issue unclosed for the devs.

--- a/docs/documentation/development/ide.md
+++ b/docs/documentation/development/ide.md
@@ -28,7 +28,7 @@ Simply follow these steps:
 4. Expand the "SmartHome" entry, double click "Runtime" and select "Next":
 ![Step 2](images/ide2.png)
 
-5. Now provide an installation folder and your Github id (used to push your changesets to) and select "Next":
+5. Now provide an installation folder (don't use spaces in the path on Windows!) and your Github id (used to push your changesets to) and select "Next":
 ![Step 3](images/ide3.png)
 
 6. The installation will now begin when pressing "Finish".

